package.json

```diff
@@ -125,6 +125,166 @@
                 }
             }
         },
+        "debuggers": [
+            {
+                "type": "aws-sam",
+                "configurationAttributes": {
+                    "direct-invoke": {
+                        "$schema": "http://json-schema.org/draft-07/schema#",
+                        "title": "AwsSamDebuggerConfiguration",
+                        "definitions": {
+                            "jsonObject": {
+                                "title": "ReadonlyJsonObject",
+                                "additionalProperties": {
+                                    "type": [
+                                        "string",
+                                        "number",
+                                        "boolean"
+                                    ]
+                                },
+                                "type": "object"
+                            }
+                        },
+                        "additionalProperties": false,
+                        "properties": {
+                            "aws": {
+                                "title": "AWS Connection",
```

Comment:
nit: Same comment as below. `Aws Connection` seems like it would generate `AwsConnection` rather than `AWSConnection`

Reply:
See above.